styles.css

```diff
@@ -0,0 +1,148 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+  background-color: #F9F9F9;
+  }
+
+.nav-bar {
+  display: inline-flex;
+  justify-content: space-between;
+  width: 100%;
+  position: top;
+  align-items: center;
+  }
+
+  #nav1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #nav2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+  }
+
+  #search-bar {
+    width: 700px;
+  }
+
+  #search-button {
+    width:10%;
+  }
+
+#img-logo {
+  height: 60%;
+  margin-right: 10em;
+  width: 100px;
+  margin-left: 15px;
+}
+
+#create-img, #apps-img, #notifications-img {
+  width: 50px;
+  height: 60%;
+}
+
+section {
+  width: 100;
+}
```

Comment:
You should always specify the unit when declaring CSS rules that deal with measurement. e.g. `width: 100%` or `width: 100px`, etc.